src/lib/notificationBroadcasterWebsocket.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const _ = require('lodash')
+const EventEmitter = require('events').EventEmitter
+const transferDictionary = require('five-bells-shared').TransferStateDictionary
+const transferStates = transferDictionary.transferStates
+const isTransferFinalized = require('./transferUtils').isTransferFinalized
+const convertToExternalTransfer = require('../models/converters/transfers')
+  .convertToExternalTransfer
+const getFulfillment = require('../models/db/fulfillments').getFulfillment
+const convertToExternalFulfillment = require('../models/converters/fulfillments')
+  .convertToExternalFulfillment
+
+class NotificationBroadcaster extends EventEmitter {
+  constructor (log, config) {
+    super()
+    this.log = log
+    this.config = config
+  }
+
+  * sendNotifications (transfer, transaction) {
```

Comment:
@justmoon To follow up -- the updated implementation doesn't sign its notification. The idea is that the authentication is done when the websocket connection is established. Sounds good?